### DISCUSSION TITLE
How do I change the size of a strumline?

### DISCUSSION THREAD

**Discussion by Mooshie318** [CodenameCrew/CodenameEngine#135] [Q&A] [answered]
I can't fit 3 strumlines on the screen without it overlapping, so I need to change the size of the strumlines so they all fit on screen without any overlapping. How would I do that?

**Comment by NeeEoo** [comment, marked as answer]
You could do like

```hx
defaultHudZoom = 0.85;
camHUD.zoom = 0.85;
```

Though this makes the rest of the hud also zoomed out.

But if you properly want to, you have to onPostNoteGeneration iirc check note.hx
